spec/controllers/idv/in_person/address_controller_spec.rb

```diff
@@ -97,17 +97,6 @@
       expect(response).to render_template :show
     end
 
-    context 'when address1 present' do
```

Comment:
Since the flow policy stipulates that we should rely on the `before_action` `confirm_step_allowed ` and not use other `before_action`s, I removed the `before_action` `confirm_in_person_address_step_needed `.  As a result, this test is no longer relevant.

@jennyverdeyen I'd be curious to hear any thoughts you have on this!

Reply:
@jennyverdeyen Good point!  I added a new test to the address_controller_spec in [2b4aa47](https://github.com/18F/identity-idp/pull/11858/commits/2b4aa477b93c7671007bba43115972a1fd69fad6).  Let me know if that provides enough coverage or if you want me to add another test.